reference/configuration/kernel.rst

```diff
@@ -56,6 +56,10 @@ directory and rename it to something else (e.g. ``foo``).
 Root Directory
 ~~~~~~~~~~~~~~
 
+.. versionadded:: 3.3
+    The ``getRootDir()`` method is deprecated since Symfony 3.3. Use the new
```

Comment:
It is: https://github.com/symfony/symfony/pull/22315/files#diff-776debe25c0a772039d45b8d692c9dbfR8